tools/Google.Cloud.Tools.Analyzers/Google.Cloud.Tools.Analyzers.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Version>1.0.0</Version>
+    <TargetFrameworks>net452</TargetFrameworks>
```

Comment:
Once this is in, I might try updating to the 2.x Roslyn packages and target netstandard1.3... then we should be able to build on Linux with no problems.